### ISSUE TITLE
Cassandra thrift/nodetool error

### ISSUE THREAD

**Issue by mikroice90** [wireapp/wire-server#690]
Hi,

when I trying to made `sydo deploy/docker-ephemeral/run.sh` i get this error:
![image](https://user-images.githubusercontent.com/48946750/55142451-f3b31c00-513c-11e9-9d57-c3ed2f328f8a.png)

I really don't know what it is going on. I customized demo-server twice on Ubuntu and there were no problems. Now when I am using CentOS 7, I have some new issues.

Thanks for help in advice.

**Comment by fisx** [comment]
it would really help if you could paste errors as text, not as images.

**Comment by mikroice90** [comment]
Here You are:
demo_wire_cassandra          | INFO  [main] 2019-03-28 08:13:38,270 Server.java:156 - Starting listening for CQL clients on /0.0.0.0:9042 (unencrypted)...
demo_wire_cassandra          | INFO  [main] 2019-03-28 08:13:39,128 CassandraDaemon.java:556 - Not starting RPC server as requested. Use JMX (StorageService->startRPCServer()) or nodetool (enablethrift) to start it

**Comment by julialongtin** [comment]
Can you contact cassandra on the listed port? Is this causing a problem
when running the integration test?

**Comment by ChrisPenner** [comment]
@mikroice90 does this cause any actual problems for you in practice? I see the same error, but all of my tests still pass and things work as expected.

**Comment by mikroice90** [comment]
Can You still made `deploy/services/demo.sh` and after it log in to server?

**Comment by tiago-loureiro** [comment]
> Can You still made deploy/services/demo.sh and after it log in to server?

This will work fine as soon as https://github.com/wireapp/wire-server/pull/694 gets merged in. Once that is done, running both:
`./deploy/docker-ephemeral/run.sh`
and
`deploy/services/demo.sh`
will work fine.

**Comment by tiago-loureiro** [comment]
@mikroice90 `develop` now works fine. The error described:

_CassandraDaemon.java:556 - Not starting RPC server as requested. Use JMX (StorageService->startRPCServer()) or nodetool (enablethrift) to start it_

is not a problem either to run integration tests nor the demo.

**Comment by tiago-loureiro** [comment]
@mikroice90 is this still an issue? As described, the issue with cassandra is not the cause of the problem so I will close this issue.

Feel free to open a new issue with a specific error that is holding you back.
